### ISSUE TITLE
[Enhancement] enable configurable keyboard shortcut for Execute command  

### ISSUE THREAD

**Issue by valsor** [little-brother/sqlite-gui#156]
Thanks again for great app!
Some people used to and/or should work also with other Database management tools as DBeaver, SQL Server Management Studio, DataGrid etc. They can use different keyboard shortcuts for Execute command (Alt + x, F5 etc). It would be very useful to have configurable keyboard shortcut for Execute command  in sqlite-gui to make life for those a little bit easier.

**Comment by little-brother** [comment]
I completely agree with you for this feature but at this time shortcuts are hard-coded, so it requires a lot of work. The priority of this feature is significant lower tnan e.g. for "Dark theme".
So maybe one day it will be implemented, but is not too soon.

**Comment by valsor** [comment]
OK. Thank you
